_posts/guides/2019-08-27-guide_the-pbr-shader.md

```diff
@@ -25,7 +25,7 @@ To make proper use of this Guide you should:
 - Know how to create textures such as Ambient Occlusion via baking
 - Be able to create PBR textures/work with the PBR Metal/Rough workflow
 - Have either Substance Painter and/or Designer and have some experience with them  
-(Tools such as Photoshop aren't covered but hopefully this guide may help with the general process)
+(Not all tools are covered but hopefully this guide may help with the general process)
```

Comment:
Would change Substance Painter/Designer to be a recommendation rather than a requirement, since a little section is provided for doing this in Photoshop.